lib/utils.ts

```diff
@@ -15,6 +15,8 @@
  *   that if it's English-specific, it should be left out of here.
  */
 
+import {render} from "preact-render-to-string";
```

Comment:
Shit, I forgot we moved `Chat.html` to `Utils.html`. I don't want Utils to have a dependency on Preact... :s